src/users/collection/getItemsInFolderForUser.test.js

```diff
@@ -22,8 +22,13 @@ describe('Users - Collection - getItemsInFolderForUser', () => {
     })
   })
   it('should get items in a folder in a user\'s collection from its username and a `folderId`', async () => {
-    const data = await client.getItemsInFolderForUser(username, folderId)
+
```

Comment:
Please do not change that test, and add a new test at the end of the file (`should accept pagination`)

Reply:
Done